src/main/java/com/codessquad/qna/controller/PostController.java

```diff
@@ -49,33 +57,60 @@ public String getAllPosts(Model model) {
     /**
      * 매개변수로 오는 id 값을 기반으로 해당 포스트를 불러옵니다.
      * 만약 해당 유저가 없다면 CanNotFindPostException를 리턴합니다.
-     * @param id
+     * @param id Post Id
      * @param model
      * @throws CanNotFindPostException
      * @return
      */
-    @GetMapping("/questions/{id}")
+    @GetMapping("/{id}")
     public String getPost(@PathVariable Long id, Model model) {
-        try{
-            Post post = postService.getPost(id);
-            model.addAttribute("post", post);
-        }catch (CanNotFindPostException e){
-            logger.error(e.getMessage());
-        }
+        model.addAttribute("post", postService.getPost(id));
         return "qna/show";
     }
 
     /**
-     * IllegalArgumentException 을 핸들링 해주는 메소드
-     * 지금은 그냥 redirect 를 하지만 나중에는 다른 페이지로 넘겨주는 작업이 필요할듯함.
-     * @param e
+     * Post UpdateForm 으로 이동할 수 있음.
+     * 세션에 로그인 되어 있는 유저와 작성자를 비교하여 틀릴시 IllegalAccessException 을 리턴함
+     * @param id Post id
+     * @param model
+     * @throws IllegalAccessException
+     * @return
+     */
+    @GetMapping("/{id}/form")
+    public String updatePostForm(@PathVariable Long id, HttpSession httpSession, Model model) throws IllegalAccessException {
+        Post post = postService.getPost(id);
+        User sessionUser = HttpSessionUtils.getUserFromSession(httpSession);
+        if(!post.isMatchedAuthor(sessionUser)){
```

Comment:
```suggestion
        if(!post.isMatchedAuthor(sessionUser)) {
```
커밋전 정렬 해주세요~
이게 힘드시면 인텔리 제이의 자동정렬을 이용합시다.

Reply:
차라리 재정렬을 전부다 해버려야겠군요.
저도 아직 컨벤션이 익숙하지 않으니 재정렬기능을 이용하도록 하겠습니다